docs/android/codestyle.md

```diff
@@ -74,3 +74,121 @@ TODOs in code tend to be forgotten over time. When someone read them later, they
 ```bash
 // TODO Missing feature (linked issue #404)
 ```
+
+## Constants
+
+### Naming conventions
+
+We follow the [Kotlin property naming guidelines](https://kotlinlang.org/docs/coding-conventions.html#property-names).
+
+### Avoid magic numbers and strings
+
+Magic numbers or strings in code can make it difficult to understand the purpose of a value, leading to poor maintainability. Always replace magic numbers or strings with named constants.
+
+#### ❌ Don't do this
+
+```kotlin
+if (value == 42) {
+  // Do something
+}
+```
+
+In this example, it’s unclear why the value 42 is being used. At the very least, you should add a comment explaining its purpose. Even better for future evolutions, it should be defined as a constant.
```

Comment:
"future evolutions"? Do you mean future changes?

From the target audience's perspective, a question that comes up is: why is a constant better?

Reply:
```suggestion
In this example, it’s unclear why the value 42 is being used. At the very least, you should add a comment explaining its purpose. Defining it as a constant is even better because it provides a clear, descriptive name, making the code easier to read, understand, and maintain. Additionally, defining the value in one place facilitates reuse across the codebase, such as in tests or within functions, classes, or other modules. This approach simplifies future changes, as updating the constant in one location automatically propagates the change wherever it is used. It also allows you to easily find where the constant is used with the IDE, avoiding irrelevant search results like searching for "42" across the entire codebase.
```

Better?